pepper-doc/src/main/doxygen/developers/developers.md

```diff
@@ -25,6 +25,25 @@ pepper-parentModule | Is the parent project for all Pepper modules and contains
 To run Pepper only the projects pepper-lib and pepper-framework are necessary. When there is already an OSGi environment only the project pepper-framework is necessary.
 
 
+Create a runnable Pepper instance with Maven
+===
+
+To create a full functional Pepper instance go to PEPPER_HOME and run
+
+\code
+mvn clean package assembly:single install
```

Comment:
Isn't "package" and "install" duplicated? From testing "mvn clean install" should work as well.